cfy_manager/components/postgresql_server/config/patroni.conf

```diff
@@ -0,0 +1,65 @@
+scope: postgres
+namespace: /db/
+name: pg{{ manager.private_ip.replace('.', '_') }}
+restapi:
+  listen: {{ manager.private_ip }}:8008
```

Comment:
might be public_ip, but there's probably something to be decided wrt. what do "private_ip" and "public_ip" even mean in the context of db and rabbitmq nodes

Reply:
We can put into the documentation that private_ip on these hosts should be set as one that is routable by all managers and all DB nodes.

Ideally maybe we should also try to get rid of the public_ip entirely? Don't we only use it for certs?

Reply:
I'll leave this as private IP (I think rabbit might use similar?) and we can discuss what to make the semantics of private_ip and public_ip (and whether to nix one of them) separately.